setup.md

```diff
@@ -8,72 +8,156 @@ permalink: /setup/
+   ```
+   $ echo $PATH
+   ```
+
+1. Run the following command to see if there are any dependencies you need to install to complete
 the setup:
 
-```
-$ flutter doctor
-```
+   ```
+   $ flutter doctor
+   ```
+
+   This command checks your environment and displays a report to the terminal window.
+   The Dart SDK is bundled with Flutter; it is not necessary to install Dart separately.
+   Check the output carefully for other software you may need to install or further 
+   tasks to perform (shown in **bold** text).
+
+   For example:
+   ```
+    ✗ Xcode end user license agreement not signed; open Xcode or run the command 'sudo xcodebuild -license'.
+
+    ✗ Brew not installed; use this to install tools for iOS device development.
+
+
+      Download brew at http://brew.sh/.
+   ```
 
-The first time you run the `flutter` command, it will download its dependencies and compile
+   The first time you run the `flutter` command, it downloads its dependencies and compiles
 itself. Subsequent runs should be much faster.
 
+The following sections describe how to perform these tasks and finish the setup process.
+You'll see in `flutter doctor` output that if you choose to use an IDE, plugins
+are availabile for IntelliJ IDEA. See [IntelliJ Setup](/intellij-setup/)
+for the steps to install the Flutter and Dart plugins.
+
+
 Once you have installed any missing dependencies, run the `flutter doctor` command again to
 verify that you’ve set everything up correctly.
 
-## Google Analytics
-
-The Flutter tool uses Google Analytics to anonymously report feature usage statistics
+The `flutter` tool uses Google Analytics to anonymously report feature usage statistics
 and basic crash reports. This data is used to help improve Flutter tools over time.
-In order to disable reporting, type `flutter config --no-analytics`, and to display the
-current state of analytics reporting, type `flutter config`.
+Analytics is not sent on the very first run or for any runs involving `flutter config`,
+so you can opt-out of analytics before any data is sent. To disable reporting, 
+type `flutter config --no-analytics` and to display the current setting, type 
+`flutter config`. See Google's privacy policy:[www.google.com/intl/en/policies/privacy](https://www.google.com/intl/en/policies/privacy/).
+{: .alert-warning}
+
+## Editor setup
 
-Analytics is not sent on the very first run or for any runs involving `flutter config`.
-This lets users choose to opt-out of analytics without any data being sent. See Google's
-privacy policy:
-[www.google.com/intl/en/policies/privacy](https://www.google.com/intl/en/policies/privacy/).
+Using our command-line tools, you can use any editor to develop Flutter applications.
+
+We recommend using our IntelliJ plug-ins for a  [rich IDE experience](/intellij-ide/) 
+supporting editing, running, and debugging Flutter apps. See [IntelliJ Setup](/intellij-setup/)
+for detailed steps.
 
 ## iOS setup
```

Comment:
This is what it says today. 

It might seem like adding statements like this is a good idea, and will give more clarity. But it has the opposite effect: clutter and confusion. Let's keep an eye on what actual users have trouble with.